{"uid":"b157a7819268e285","name":"test_parse_arrays[None-None-date]","fullName":"tests.unit.common.test_typing_parse#test_parse_arrays","historyId":"2e839606da0eeb2a785e6773832d2a85","time":{"start":1729134655183,"stop":1729134655184,"duration":1},"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","statusTrace":"self = <allure_pytest.listener.AllureListener object at 0x7f363b729520>\nfixturedef = <FixtureDef argname='type' scope='function' baseid=''>\n\n    @pytest.hookimpl(hookwrapper=True)\n    def pytest_fixture_post_finalizer(self, fixturedef):\n        yield\n        if hasattr(fixturedef, 'cached_result') and self._cache.get(fixturedef):\n            container_uuid = self._cache.pop(fixturedef)\n>           self.allure_logger.stop_group(container_uuid, stop=now())\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_pytest/listener.py:190: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_commons/reporter.py:91: in stop_group\n    plugin_manager.hook.report_container(container=group)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_commons/logger.py:34: in report_container\n    self._report_item(container)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/allure_commons/logger.py:25: in _report_item\n    with io.open(self._report_dir / filename, 'w', encoding='utf8') as json_file:\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_io.py:93: in open\n    return fake_open(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:105: in fake_open\n    return fake_file_open(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:136: in __call__\n    return self.call(*args, **kwargs)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:244: in call\n    file_object = self._init_file_object(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_open.py:358: in _init_file_object\n    file_object = self.filesystem.create_file_internally(\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_filesystem.py:2551: in create_file_internally\n    self.raise_os_error(errno.ENOENT, parent_directory)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pyfakefs.fake_filesystem.FakeFilesystem object at 0x7f3638de4400>\nerr_no = 2\nfilename = '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'\nwinerror = None\n\n    def raise_os_error(\n        self,\n        err_no: int,\n        filename: Optional[AnyString] = None,\n        winerror: Optional[int] = None,\n    ) -> NoReturn:\n        \"\"\"Raises OSError.\n        The error message is constructed from the given error code and shall\n        start with the error string issued in the real system.\n        Note: this is not true under Windows if winerror is given - in this\n        case a localized message specific to winerror will be shown in the\n        real file system.\n    \n        Args:\n            err_no: A numeric error code from the C variable errno.\n            filename: The name of the affected file, if any.\n            winerror: Windows only - the specific Windows error code.\n        \"\"\"\n        message = os.strerror(err_no) + \" in the fake filesystem\"\n        if winerror is not None and sys.platform == \"win32\" and self.is_windows_fs:\n            raise OSError(err_no, message, filename, winerror)\n>       raise OSError(err_no, message, filename)\nE       FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pyfakefs/fake_filesystem.py:489: FileNotFoundError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"global_fake_fs","time":{"start":1729134655176,"stop":1729134655182,"duration":6},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"clear_cache","time":{"start":1729134655176,"stop":1729134655176,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"global_fake_fs::0","time":{"start":1729134655191,"stop":1729134655191,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"parentSuite","value":"tests.unit.common"},{"name":"suite","value":"test_typing_parse"},{"name":"host","value":"fv-az1393-657"},{"name":"thread","value":"2163-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.unit.common.test_typing_parse"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"expected","value":"None"},{"name":"type","value":"<class 'datetime.date'>"},{"name":"value","value":"None"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":18,"skipped":0,"passed":58,"unknown":0,"total":76},"items":[{"uid":"6da11c842a6232e2","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/330//#testresult/6da11c842a6232e2","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729134519731,"stop":1729134519731,"duration":0}},{"uid":"8e3d91a2fbf471c8","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/326//#testresult/8e3d91a2fbf471c8","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729048373827,"stop":1729048373827,"duration":0}},{"uid":"60ea35a28c1433fa","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/329//#testresult/60ea35a28c1433fa","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1729048221586,"stop":1729048221587,"duration":1}},{"uid":"cd157d9caad2786c","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/325//#testresult/cd157d9caad2786c","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1728961918748,"stop":1728961918749,"duration":1}},{"uid":"8fb7743c0978b4f5","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/328//#testresult/8fb7743c0978b4f5","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1728961776575,"stop":1728961776576,"duration":1}},{"uid":"a221c81fb0fead83","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/324//#testresult/a221c81fb0fead83","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1728875624864,"stop":1728875624865,"duration":1}},{"uid":"8c51377eacab821b","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/327//#testresult/8c51377eacab821b","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1728875470197,"stop":1728875470197,"duration":0}},{"uid":"d713dddb8917b650","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/323//#testresult/d713dddb8917b650","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1728789188455,"stop":1728789188455,"duration":0}},{"uid":"1d66af4e727839cf","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/326//#testresult/1d66af4e727839cf","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1728789057766,"stop":1728789057767,"duration":1}},{"uid":"c0e6b24cec20c8ac","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/322//#testresult/c0e6b24cec20c8ac","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1728702506889,"stop":1728702506890,"duration":1}},{"uid":"54e7003cae204ac9","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/325//#testresult/54e7003cae204ac9","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1728702389110,"stop":1728702389110,"duration":0}},{"uid":"5f54919da52e13d3","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/321//#testresult/5f54919da52e13d3","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1728616194099,"stop":1728616194099,"duration":0}},{"uid":"332ba29a66723032","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/324//#testresult/332ba29a66723032","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1728616056400,"stop":1728616056400,"duration":0}},{"uid":"f094ab53cd228a14","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/320//#testresult/f094ab53cd228a14","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1728530075329,"stop":1728530075329,"duration":0}},{"uid":"ad083469f75c2138","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/323//#testresult/ad083469f75c2138","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1728529708772,"stop":1728529708772,"duration":0}},{"uid":"6c3c7c5b9c1fcc19","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/319//#testresult/6c3c7c5b9c1fcc19","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1728443438630,"stop":1728443438630,"duration":0}},{"uid":"8fd305b61255676b","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/322//#testresult/8fd305b61255676b","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory in the fake filesystem: '/home/runner/work/firebolt-python-sdk/firebolt-python-sdk/allure-results'","time":{"start":1728443306497,"stop":1728443306497,"duration":0}},{"uid":"6eb0b08825eafe03","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/318//#testresult/6eb0b08825eafe03","status":"passed","time":{"start":1728357112323,"stop":1728357112323,"duration":0}},{"uid":"785b9389279e3895","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/321//#testresult/785b9389279e3895","status":"passed","time":{"start":1728356979085,"stop":1728356979086,"duration":1}},{"uid":"a39b71d5d2288959","reportUrl":"https://firebolt-db.github.io/firebolt-python-sdk/_2bddcd39486ecb4e8d7d7debea7fd95bd3aab741_unit/317//#testresult/a39b71d5d2288959","status":"passed","time":{"start":1728270718140,"stop":1728270718141,"duration":1}}]},"tags":[]},"source":"b157a7819268e285.json","parameterValues":["None","<class 'datetime.date'>","None"]}